_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,268 @@
+
+For additional information and future developments towards better integration with KServe, you can refer to the slides
+[here](https://docs.google.com/presentation/d/1wprxN0n23EMkPRX_PaZZcIzZbn_i8Sh_).
+
+## Training Operator (Trainer) & Katib
+
+Kubeflow 1.10 enhances the Training Operator and Katib, providing new tools and APIs for hyperparameter optimization,
+particularly for large language models.
+
+### Hyperparameter Optimization API for LLMs
+
+Kubeflow 1.10 introduces a new high-level [API for hyperparameter tuning](https://github.com/kubeflow/katib/issues/2339),
+streamlining LLMOps workflows in Kubernetes. This API integrates Katib and the Training Operator to automate
+hyperparameter optimization, reducing manual effort for data scientists fine-tuning large language models.
+
+### Support for Various Parameter Distributions
+
+Kubeflow 1.10 enhances Katib’s hyperparameter optimization capabilities by introducing
+[support for multiple probability distributions](https://github.com/kubeflow/katib/issues/2374). Previously limited to
+uniform distributions, Katib now supports log-uniform, normal, and log-normal distributions, providing data scientists
+with greater flexibility in tuning hyperparameters. This is particularly useful for parameters like learning rates,
+which benefit from log-uniform sampling, or values expected to vary around a mean, suited for normal distributions.
+
+### Push-Based Metrics Collection
+
+Katib now allows users to push metrics to Katib DB directly.The new push-based design provides administrative and
+performanace improvements to the existing pull based design.
+
+## Dashboard & Notebooks
+
+* Prometheus Metrics Enhancement: Notebooks and CRUD backends now expose Prometheus metrics for improved observability.
+* OAuth2-Proxy Logout Fix: Resolving logout issues for proper user logout.
+* Improved Notebook Creation Error Messages: Providing more specific error messages for resource constraint failures.
+* Update all Notebook / Workspace images and migrate most images to GHCR.
+* For more information, please consult the [release notes](https://github.com/kubeflow/kubeflow/releases/tag/v1.10.0).
+
+Kubeflow 1.10 improves the observability and usability of Notebooks, while providing updated
+[default images](https://github.com/kubeflow/kubeflow/pull/7687).
+
+### Prometheus Metrics for Notebooks
+
+Both the Notebooks component and CRUD backends now feature Prometheus metrics. Notebooks expose custom metrics using the
+prom-client library, and CRUD backends utilize the prometheus_flask_exporter library. This ensures consistent metrics
+integration across all backend services.
+
+### More Descriptive Error Messages
+
+Error messages for notebook creation failures due to resource constraints are now more descriptive. Users can quickly
+identify issues such as insufficient resources.
+
+## Spark Operator
+
+Details about new releases since 1.9:
+...
+
+Benchmarking was done on Kubeflow Spark Operator:
```

Comment:
It would be nice to highlight some results from this blog post here (e.g. number of Spark Applications and Kubernetes pods). @kubeflow/wg-data-leads @vara-bonthu @nabuskey Any thoughts ?
Btw, you can directly link the blog post: https://blog.kubeflow.org/operators/benchmarking/performance/2025/03/15/kubeflow-spark-operator-benchmarks.html